### PR TITLE
chore: [IOBP-696] Add payments section banner definitions

### PR DESCRIPTION
**Pull request by Hantex9**
## Short description
This PR adds definitions and an initial implementation of the alert (formally called a banner) service to be displayed on the home payments page.

In this PR @thisisjp will insert the texts to be shown for this banner.

## List of changes proposed in this pull request
- Added definitions
- Upgraded package version
- Added a scaffolding structure for the payments banner section

### CODE REVIEW

**Comment by thisisjp** [comment]
@Hantex9 the placeholder text is fine, we'll change it and turn it on when we'll have a critical issue (🤞 let's hope not )

File: status/backend.json [hunk 1]
```diff
@@ -211,6 +211,14 @@
         "en-EN": "pagoPA is currently not available. Our technicians are working to fix the issue."
       }
     },
+    "payments": {
+      "is_visible": false,
+      "level": "critical",
```
Comment (thisisjp) [inline status/backend.json:216]:
```suggestion
      "level": "normal",
```

File: status/backend.json [hunk 2]
```diff
@@ -211,6 +211,14 @@
...
+      "message": {
+        "it-IT": "pagoPA non è al momento disponibile. I nostri tecnici sono al lavoro per risolvere il problema.",
```
Comment (thisisjp) [inline status/backend.json:218]:
```suggestion
        "it-IT": "Stiamo riorganizzando i metodi di pagamento salvati in app.",
```

File: status/backend.json [hunk 3]
```diff
@@ -211,6 +211,14 @@
...
+        "en-EN": "pagoPA is currently not available. Our technicians are working to fix the issue."
```
Comment (thisisjp) [inline status/backend.json:219]:
```suggestion
        "en-EN": "We are rearranging the payment methods saved in the app."
```

File: status/backend.json [hunk 4]
```diff
@@ -211,6 +211,14 @@
         "en-EN": "pagoPA is currently not available. Our technicians are working to fix the issue."
       }
     },
+    "payments": {
+      "is_visible": false,
+      "level": "normal",
```
Comment (thisisjp) [inline status/backend.json:216]:
```suggestion
      "level": "normal",
      "web_url": {
        "it-IT": "https://io.italia.it/nuova-sezione-pagamenti",
        "en-EN": "https://io.italia.it/nuova-sezione-pagamenti"
      },
```

**Review by thisisjp — APPROVED** [review]

**Review by mastro993 — APPROVED** [review]

**Pull request merged**